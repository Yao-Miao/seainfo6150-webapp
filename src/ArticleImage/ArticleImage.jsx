import React from "react";
import classes from "./ArticleImage.module.css";

const ArticleImage = (props) => {
  return (
    <img className={classes.Image} src={props.url} alt={props.title}/>
  );
};

export default ArticleImage;
