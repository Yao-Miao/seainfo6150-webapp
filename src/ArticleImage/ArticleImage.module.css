.Image{
    border: solid;
    border-color: red;
    border-width: 4px;
    border-radius:50%;
    width: 100px;
    height: 70px;
}