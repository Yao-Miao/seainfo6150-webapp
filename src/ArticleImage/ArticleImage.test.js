import React from "react";
import ArticleImage from "./ArticleImage";
import { fireEvent } from "@testing-library/react";

describe("ArticleListItem tests", () => {
  it("renders correctly", () => {
    const title =
      "ArticleListItem.jsx:6 The Statue of Liberty's torch heads to new museum";
    const url =
      "https://cdn.cnn.com/cnnnext/dam/assets/181122102925-statue-of-liberty-crown-super-169.jpg";

    const { container } = render(<ArticleImage url={url} title={title} />);

    expect(container).toMatchSnapshot();
  });
});
