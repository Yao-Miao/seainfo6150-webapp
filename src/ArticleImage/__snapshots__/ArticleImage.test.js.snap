// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleListItem tests renders correctly 1`] = `
<div>
  <img
    alt="ArticleListItem.jsx:6 The Statue of Liberty's torch heads to new museum"
    class="Image"
    src="https://cdn.cnn.com/cnnnext/dam/assets/181122102925-statue-of-liberty-crown-super-169.jpg"
  />
</div>
`;
